app/services/rand_service.rb

```diff
@@ -0,0 +1,12 @@
+class RandService
+  ALPHA_CHARSET = Array('A'..'Z') + Array('a'..'z')
+  UPCASE_ALHPA_CHARSET = Array('A'..'Z')
+
+  def self.upcase_alpha(size)
+    Array.new(size) { UPCASE_ALHPA_CHARSET.sample }.join
+  end
+
+  def self.alpha(size)
+    Array.new(size) { ALHPA_CHARSET.sample }.join
```

Comment:
@bsharpe I don't think you're actually using this method, but in any case, there's a typo here, leading to the wrong constant. There's a typo in `UPCASE_ALPHA_CHARSET` too.

Reply:
gracias!